Add documentation for PRs and Auto Insights (#2)
Codesee-io/docs-maps: docs/guide.md

The file browser also includes a search bar at the top, where you can filter by the name of files or folders.

### Auto insights

> Once your map is created, CodeSee will automatically gather some insights into your repo and display them for you in an easy-to-understand way!

Reviewer: ```suggestion
CodeSee will automatically gather insights from your repository. Use the toggle switches to enable or disable the insights on your Map. 
```

> Your labels will be temporarily replaced by those corresponding to the insights themselves, but don't worry! All you have to do is disable the insight to restore yours!

Reviewer: ```suggestion
When Insights are displayed on your Map, your labels will be temporarily replaced by those corresponding to the insights themselves. To restore your labels, disable the Insight using the toggle switch.
```

![Screenshot of CodeSee's insights, showing the engineering hot spots insights](img/maps-insights.png)

Below the File Browser, you'll find three insight types:

> - **Engineering hot spots**: See how which files and folders have the most recent commits

Reviewer: ```suggestion
- **Engineering Hot Spots**: See which files and folders have had the most recent commits.
```

> - **Latest activity**: See when the last commit to a file or folder was made

Reviewer: ```suggestion
- **Latest Activity**: See when the last commit to a file or folder was made.
```

> - **Creation date**: See when the files/folders were created, relative to today's date

Reviewer: ```suggestion
- **Creation Date**: See when files and folders were created relative to today's date.
```
